Dockerfile.cpu

```diff
@@ -25,87 +24,48 @@ RUN apt-get update \
 		openssh-client \
 		openssh-server \
 		pkg-config \
-		wget \
-	&& rm -rf /var/lib/apt/lists/* \
-	&& rm /etc/ssh/ssh_host_ecdsa_key \
-	&& rm /etc/ssh/ssh_host_ed25519_key \
-	&& rm /etc/ssh/ssh_host_rsa_key
+		wget
 
-ARG CONDA_DIR="/opt/conda"
-ARG CONDA_INSTALLER="Miniconda3-4.7.10-Linux-x86_64.sh"
-ARG CONDA_MD5="1c945f2b3335c7b2b15130b1b2dc5cf4"
-ARG CONDA_URL="https://repo.continuum.io/miniconda"
-ENV CONDA_DIR="${CONDA_DIR}" PATH="${CONDA_DIR}/bin:${PATH}"
+COPY docker_files /tmp/det_docker_files
```

Comment:
I'm not a big fan of the name docker_files. dockerfile_helpers or something might be better?

Reply:
Why not just use the existing `files` directory?  Seems not terribly different from this new `det_docker_files` directory to me.

Reply:
In fact the previous contents of files/ just got moved under here. I went with dockerfile_scripts. I feel like that's clearer - we had top-level directories named "files" and "scripts", and it's not terribly clear one is used in Dockerfiles and one is used for publishing the images... Thoughts?

Reply:
I don't care very much, tbh